### PR TITLE
Tidy up the sidebar, add link to related docs

### PR DESCRIPTION
**Pull request by timcowlishaw**
## Changes in this PR:

This does a few things

- add a link to related documents in the editor sidebar
- refresh the visual design of the sidebar per terry's latest designs
- refactor the logic for identifying document type and related document in order to reduce repetition
- Bumps caselawapiclient to 13.1.0 and takes advantage of the new document type and document existence querying logic therein

## Trello card / Rollbar error (etc)

https://trello.com/c/PfJOSaeH/1202-eui-ps-add-a-link-to-the-associated-document-on-the-side-panel-for-publish-and-put-on-hold-screens

### Before
Screenshot 2023-07-26 at 18 45 36

### After

Screenshot 2023-07-26 at 18 41 35

### CODE REVIEW

**Comment by timcowlishaw** [comment]
[EUI: PS - add a link to the associated document on the side panel for 'publish and 'put on-hold' screens.](https://trello.com/c/PfJOSaeH/1202-eui-ps-add-a-link-to-the-associated-document-on-the-side-panel-for-publish-and-put-on-hold-screens)

**Review by Terry-Price — CHANGES_REQUESTED** [review]
Looks a whole lot better @timcowlishaw 
Was just wondering two small things that might help:

- Could the bold headings line height match the link line height so the clusters were uniform line height. might make it visually simpler.
- Was wondering about the word 'problem' could it be 'issue' as this is seen as surmountable  and a little softer business language.

**Comment by timcowlishaw** [comment]
hey @Terry-Price aha no probs - how's this? That copy change was already done, i just forgot to do `compilemessages` before taking the screenshot!
Screenshot 2023-07-27 at 16 23 08

File: judgments/utils/__init__.py [hunk 1]
```diff
@@ -171,3 +173,29 @@ def get_judgment_by_uri(judgment_uri: str) -> Judgment:
     )
 
     return Judgment(judgment_uri, api_client)
+
+
+def check_document_at_uri_exists(document_uri: str) -> Optional[str]:
```
Comment (jacksonj04) [inline judgments/utils/__init__.py:178]: There's already a function in the API client for this; `MarklogicApiClient.document_exists(str: uri)` will give you a boolean for if there's a document at the given URI.

**Review by jacksonj04 — CHANGES_REQUESTED** [review]
There's a whole bunch of test changes you can avoid by swapping to an already-existing API function over writing your own document existence check.

File: judgments/utils/__init__.py [hunk 2]
```diff
@@ -171,3 +173,29 @@ def get_judgment_by_uri(judgment_uri: str) -> Judgment:
...
+    try:
+        get_judgment_by_uri(document_uri)
+        return document_uri
+    except DocumentNotFoundError:
+        return None
+
+
+def set_document_type_and_link(
```
Comment (anthonyhashemi) [inline judgments/utils/__init__.py:186]: You can now update EUI to use apiclient 13.0.0 or above to determine the document type from the document_uri.

Comment (anthonyhashemi) [reply judgments/utils/__init__.py:186]: ah I see Nick already beat me to that comment

File: judgments/utils/__init__.py (hunk 1 above)
Comment (timcowlishaw) [reply judgments/utils/__init__.py:178]: aha ace, thank you Nick!

File: judgments/utils/__init__.py (hunk 2 above)
Comment (timcowlishaw) [reply judgments/utils/__init__.py:186]: perfect, that simplifies things considerably!

**Review by dragon-dxw — APPROVED** [review]

**Review by Terry-Price — APPROVED** [review]
Spiffingly good.

**Review by jacksonj04 — APPROVED** [review]

**Pull request merged**